taxonium_component/src/hooks/useLayers.jsx

```diff
@@ -540,6 +545,49 @@ const useLayers = ({
   layers.push(minimap_line_horiz, minimap_line_vert, minimap_scatter);
   layers.push(minimap_bound_polygon);
 
+  let countryCounts = {};
```

Comment:
It's not clear to me that we will end up using much of this logic other than the layers.push bit. E.g. the `max` and `min` aren't used at present. I would probably suggest removing the color piece as it may never be used and leaving all countries coloured the same for now.

Reply:
Removed the color code for now.